src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/csip/SyncTransformer.kt

```diff
@@ -114,7 +114,14 @@ fun CSIPResponse.toActionDetails() = getActionDetails(
   lastModifiedBy,
   lastModifiedByDisplayName,
 )
-
+fun Attendee.toActionDetails() = getActionDetails(
```

Comment:
There is no inheritance here so couldn't figure out a neat solution apart from all calling the same getter

Reply:
I suppose you could add an interface to all JPA objects with the audit columns  but not sure I would bother